conda-forge.yml

```
@@ -1,3 +1,7 @@
 conda_forge_output_validation: true
 bot:
   automerge: true
+channels:
+  sources:
+    - conda-forge
+    - defaults
```

```suggestion
channels:
  sources:
    - conda-forge
    - defaults
build_with_mambabuild: False
```
Also, try using conda-build to see if it respects the channels parameter.

---

Sadly it looks like this didn't work.

---

Any changes to the conda-forge.yml require a rerender.